src/Formats/FormatSettings.h

```diff
@@ -270,6 +270,8 @@ struct FormatSettings
         UInt64 max_column_pad_width = 250;
         UInt64 max_value_width = 10000;
         bool color = true;
+	
+		SettingFieldUInt64Auto output_format_pretty_color{"auto"};
```

Comment:
No need to add a new setting for it. We can just change the type of `color` setting.